connector/rest/src/main/java/it/pagopa/selfcare/onboarding/connector/PartyConnectorImpl.java

```diff
@@ -161,7 +161,7 @@ public void onboardingOrganization(OnboardingData onboardingData) {
     }
 
     @Override
-    public Collection<InstitutionInfo> getOnBoardedInstitutions(Set<String> productFilter) {
+    public Collection<InstitutionInfo> getOnBoardedInstitutions(String productFilter) {
         log.trace("getOnBoardedInstitutions start");
         OnBoardingInfo onBoardingInfo = restClient.getOnBoardingInfo(null, EnumSet.of(ACTIVE));
         Collection<InstitutionInfo> result = parseOnBoardingInfo(onBoardingInfo, productFilter);
```

Comment:
are we retrieving all institutions onboarded and then we apply filter on product? 

pay attention at instituions size, could be better to apply filter on ms-core